Sem2_Homework/task3.py

```diff
@@ -0,0 +1,5 @@
+# Задайте список из n чисел последовательности (1 + 1 / n) ** n и выведите на экран их сумму.
+
+n = int(input("Введите количество чисел:  "))
+
+print(sum([(1 + 1 / x) ** x for x in range(1, n+1)]))

```

Comment:
```suggestion
print(sum((1 + 1 / x) ** x for x in range(1, n+1)))

```
Не нужно делать генератор списков, функция `sum` и так поймет.